### PR TITLE
[IntelliJ Plugin] Add plugin fixes

### LINKED ISSUE #15087: Syntax highlighting issue in IntelliJ IDEA
**Description:**

There seems to be an issue in syntax highlighting (from line 69) 
image

**Steps to reproduce:**
-
**Affected Versions:**
Ballerina 0.990.2

**OS, DB, other environment details and versions:**
IDE - IntelliJ IDEA Ultimate 

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
This PR adds fixes related to
- latest xml literal syntax changes 
- latest string literal syntax changes
- **checkpanic** highlighting issue

Fixes #15087, #13872.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: Shall we update the year? Need to do this in the template.

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/BallerinaParser.g4 [hunk 2]
```diff
@@ -522,7 +549,16 @@ returnStatement
     ;
 
 workerSendAsyncStatement
-    :   expression RARROW Identifier (COMMA expression)? SEMICOLON
+    :   expression RARROW peerWorker (COMMA expression)? SEMICOLON
+    ;
+
+peerWorker
+    : workerName
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/BallerinaParser.g4]: Need to fix the indentation.

File: tool-plugins/intellij/src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -39,8 +39,10 @@
         <completion.contributor implementationClass="org.wso2.lsp4intellij.contributors.LSPCompletionContributor"
                                 id="LSPCompletionContributor" language="any"/>
 
-        <inspectionToolProvider implementation="org.wso2.lsp4intellij.contributors.inspection.LSPInspectionProvider"
-                                id="LSPInspectionProvider"/>
+        <localInspection displayName="LSP"
+                         shortName="LSP"
+                         enabledByDefault="true"
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/resources/META-INF/plugin.xml]: Shall we inline these 3 lines?

File: tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java (hunk 1 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: I've already changed the template and works fine when adding new classes. But not being applied for the generated files :/ Strange

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/BallerinaParser.g4 (hunk 2 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/BallerinaParser.g4]: Fixed

File: tool-plugins/intellij/src/main/resources/META-INF/plugin.xml (hunk 3 above)
Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/resources/META-INF/plugin.xml]: Formatted accordingly

File: tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java (hunk 1 above)
Comment (Shan1024) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: Did you try after deleting all the generated files?

Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: Yeah I did. Tried with restarting the IDE as well.

Comment (Shan1024) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: I think you might have edited a wrong template. Can you let me know what is the template you edited?

Comment (Shan1024) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: You need to update the following file and regenerate the classes - https://github.com/ballerina-platform/ballerina-lang/blob/master/tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/CopyrightHeader.txt

Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaCheckPanicExpression.java]: Oh my bad :/ I tried changing the file header in IDEA file templates.

Fixed :) Thanks for pointing out the fix.

**Pull request merged**